common/djangoapps/cme_registration_v2/cme_reg_form/admin.py

```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from .models import ExtraInfo
+
+admin.site.register(ExtraInfo)
```

Comment:
I think it is only required if we need to fiddle with the model in django admin which I don't think we will.